src/backend/aspen/workflows/nextstrain_run/builds_templates/group_plus_context.yaml

```diff
@@ -23,8 +23,8 @@ custom_rules:
 files:
   lat_longs: "my_profiles/aspen/lat_longs.tsv"
   ordering: "my_profiles/aspen/ordering.tsv"
-  description: "my_profiles/aspen/covidtrackerca.md"
```

Comment:
Can we delete this file if it's not going to be used anymore? Should we also update group.yaml?

Reply:
group.yaml will no longer be used after on demand release, right? we can then delete:
group.yaml
aspen_auspice_config.json
covidtrackerca.md